build/makelib/golang.mk

```diff
@@ -173,7 +173,12 @@ go.vendor: $(DEP)
 $(DEP):
 	@echo === installing dep
 	@mkdir -p $(TOOLS_HOST_DIR)
-	@curl -sL -o $(DEP) https://github.com/golang/dep/releases/download/$(DEP_VERSION)/dep-$(GOHOSTOS)-$(GOHOSTARCH)
+	@if [ "$(GOHOSTARCH)" = "arm64" ]; then\
+		go get -u github.com/golang/dep/cmd/dep;\
```

Comment:
please use `@GOPATH=$(TOOLS_HOST_DIR)/tmp GOBIN=$(TOOLS_HOST_DIR) $(GOHOST) get ...` to avoid tainting of the host go path.

Reply:
Thanks for the review! Done and will push the update soon.